tests/ecp5/dpram.ys

```diff
@@ -0,0 +1,17 @@
+read_verilog dpram.v
+hierarchy -top top
+proc
+memory -nomap
+equiv_opt -run :prove -map +/ecp5/cells_sim.v synth_ecp5
+memory
+opt -full
+
+miter -equiv -flatten -make_assert -make_outputs gold gate miter
+
+#ERROR: Failed to import cell gate.mem.0.0.0 (type DP16KD) to SAT database.
+#sat -verify -prove-asserts -seq 3 -set-init-zero -show-inputs -show-outputs miter
```

Comment:
TODO

Reply:
This is because we don't have a Verilog model for the `DP16KD` yet...

Reply:
I think the decision is to temporarily disable this test for now.

@SergeyDegtyar can you create a new ticket to keep track of the need to add a DP16KD model before it can re-enabled, please?

Reply:
I've created  #1358 as a more general tracking issue for this.

Reply:
@SergeyDegtyar Can you leave a comment in the testcase indicating that it is blocked by #1358 please?

Reply:
OK. Already added this comment.